lib/formatters/pretty.js

```diff
@@ -99,7 +102,10 @@ class PrettyPrinter {
       .map((error) => PrettyPrinter._formatError(error, options))
       .join('\n');
 
-    return `${chalk.underline(filePath)}\n${errorsMessages}\n`;
+    let displayFilePath =
+      ciInfo.isCI && options.workingDir ? path.join(options.workingDir, filePath) : filePath;
```

Comment:
Is workingDir a resolved/absolute path?

Reply:
When we default it, it is but not when it is not defaulted.

https://github.com/ember-template-lint/ember-template-lint/blob/6ec1d058c31e1c323cf7937ace4beb83b63780c2/bin/ember-template-lint.js#L220-L222

Will fix to ensure it is always absolute.

Reply:
Not sure that it is blocking, but I think it might be nice to allow this to be configured as an option to show absolute paths. Personally as a dev used to eslint, dotnet build warnings, PHP warnings, and TS errors I'm more used to having absolute paths in local dev too so I disagree with the assumption of "seeing the relative path looks better"

Reply:
I think it's less about it "looking better" but rather it's a bit "easier on the eyes". I do think you have a point, and consistency across tools can be a strong driver for these types of features.